src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -195,7 +195,7 @@ await window.getScreenDetails();
     isExtended: true
     isInternal: true
     isPrimary: true
-    label: "Internal Display 1"
+    label: "Built-in Retina Display"
```

Comment:
Out of curiosity, do you know why I get "Built-in Retina Display" and not "Built-in Liquid Retina XDR Display" as in macOS settings. I wonder if they're using something else than [localizedname](https://developer.apple.com/documentation/appkit/nsscreen/3228043-localizedname?language=objc) @michaelwasserman 

<img width="698" alt="image" src="https://user-images.githubusercontent.com/634478/179527973-a8d58c64-8b84-4d4e-b701-a86358d373c9.png">

Reply:
That's a good question. I don't see any other obvious NSScreen properties to use, perhaps Apple generates more detailed names given other display configuration data? Maybe @bradtriebwasser would know more?

Reply:
Thanks!